app/controllers/concerns/remotely_translatable.rb

```diff
@@ -16,12 +16,16 @@ def remote_translation_for(resource)
     end
 
     def translation_empty?(resource)
-      resource.translations.where(locale: I18n.locale).empty?
+      resource.class.translates? && resource.translations.where(locale: I18n.locale).empty?
```

Comment:
Would it be possible to add a test case for this change?

Reply:
Solved in the commit: 18900da